discord/guild.py

```diff
@@ -1296,6 +1296,11 @@ async def prune_members(self, *, days, compute_prune_count=True, reason=None):
         To check how many members you would prune without actually pruning,
         see the :meth:`estimate_pruned_members` function.
 
+        To prune members that have specific roles see the :meth:`roles` function.
```

Comment:
```suggestion
        To prune members that have specific roles see the ``roles`` parameter.
```
This could be removed outright.

Reply:
Oh yeah, my bad on that one, I read the above one and mistook it for compute_prune_count for some reason, lol.